First/src/firstPackage/Example2.java

```diff
@@ -0,0 +1,32 @@
+package firstPackage;
+
+public class Example2 {
+
+	public static void main(String[] args) {
+		String s = "aaabb";
```

Comment:
Same here

Reply:
Done !! Added new files named as EqualOccurance.java

Reply:
remove this Example2 file

Reply:
Done ! removed  Example2 file